src/components/ui/breadcrumb.tsx

```
@@ -0,0 +1,122 @@
+}
+
+function BreadcrumbPage({ className, ...props }: React.ComponentProps<"span">) {
+  return (
+    <span
+      data-slot="breadcrumb-page"
+      role="link"
+      aria-disabled="true"
+      aria-current="page"
+      className={cn("font-normal text-foreground", className)}
+      {...props}
+    />
```

Biome (2.4.13)

[error] The HTML element with the interactive role "link" is not focusable.

(lint/a11y/useFocusableInteractive)